requirements.txt

```diff
@@ -4,7 +4,7 @@ bottle==0.12.25
 cheroot==10.0.0
 gpsdclient==1.3.2
 luma.oled==3.12.0
-pillow==9.5.0
+pillow
```

Comment:
Was 9.5.0 not working in some way?  If not, what version did pip end up pulling for you so we can add it here (`pip list`)?  I like to keep the requirements file version locked just to avoid any mysterious issues from one image build to the next.